<launch>
  <!-- Name of the node -->
  <arg name="controller_node" default="bur_controller"/>

  <node pkg="controller" name="$(var controller_node)" exec="controller" output="screen">
    <param name="sub_topic" value="command"/>
    <param name="target_vel_topic" value="/cmd_vel"/>
    <param name="pub_topic" value="control_effort"/>

    <param name="use_command_target" value="true"/>

    <param name="linear_x/p" value="2.0" type="float"/>
    <param name="linear_x/i" value="0.0" type="float"/>
    <param name="linear_x/d" value="0.0" type="float"/>

    <param name="linear_y/p" value="2.0" type="float"/>
    <param name="linear_y/i" value="0.0" type="float"/>
    <param name="linear_y/d" value="0.0" type="float"/>

    <param name="linear_z/p" value="10.0" type="float"/>
    <param name="linear_z/i" value="5.0" type="float"/>
    <param name="linear_z/d" value="5.0" type="float"/>

    <param name="angular_x/p" value="1.0" type="float"/>
    <param name="angular_x/i" value="0.0" type="float"/>
    <param name="angular_x/d" value="0.0" type="float"/>

    <param name="angular_y/p" value="1.0" type="float"/>
    <param name="angular_y/i" value="0.0" type="float"/>
    <param name="angular_y/d" value="0.0" type="float"/>

    <param name="angular_z/p" value="3.0" type="float"/>
    <param name="angular_z/i" value="0.0" type="float"/>
    <param name="angular_z/d" value="1.0" type="float"/>
    <!-- <param name="angular_z/p" value="10.0" type="float"/>
    <param name="angular_z/i" value="0.0" type="float"/>
    <param name="angular_z/d" value="5.0" type="float"/> -->
  </node>
</launch>
